### PR TITLE
Orders Page UI Improvements

### PR DESCRIPTION
**Pull request by newlinewebdesign**

### CODE REVIEW

File: plugins/j2store/report_products/report_products/tmpl/default.php [hunk 1]
```diff
@@ -179,7 +179,7 @@
                             <tbody>
                             <tr>
                                 <td><?php echo $product->orderitem_name; ?>
-                                    <br>
+                                    <br/>
```
Comment (brianteeman) [inline plugins/j2store/report_products/report_products/tmpl/default.php:182]: Please revert this change. Joomla uses html where the self closing slash is not used. This is xhtml

Comment (newlinewebdesign) [reply plugins/j2store/report_products/report_products/tmpl/default.php:182]: I only touched the language string as it was hijacking a core j2store language string (J2STORE_FILTER_SEARCH) but I must not have synced the branch so reverting now.
